ops/fhir_validations/validations_ignorelist.yml

```diff
@@ -49,3 +49,7 @@ ignore_list:
       error_patterns:
         - ".*ExplanationOfBenefit\\.diagnosis\\.diagnosis\\[x\\].* minimum required.*"  # Missing field (6 errors)
         - ".*ReferralRequest\\.intent.* minimum required.*"                             # Missing field (8 errors)
+    # V2 specific validation issues
```

Comment:
in V1, Eob.referral supplied a #reference to a contained ReferralRequest resource.  In FHIR R4, the Referral Request resource was renamed/refactored to ServiceRequest and in V2 we changed our mapping.  Eob.referral now supplies a logical reference e.g. a provider identifier (as shown below) instead of a reference to another resource.   Having said all that, I'm still confused as to why the ignore entry in the yml file was needed for V2, but Chris said it resolved the error and I don't think it causes any harm to have it.

` "identifier" : {`
`      "type" : {`
`        "coding" : [ {`
`          "system" : "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBIdentifierType",`
`          "code" : "npi",`
`          "display" : "National Provider Identifier"`
`        } ]`
`      },`
`      "value" : "8765676"`

Reply:
added